ajax/libs/CSS-Mint/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "CSS-Mint",
+  "filename": "css-mint.min.css",
+  "description": "Lightweight and simple to use UI Kit",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ArunMichaelDsouza/CSS-Mint.git"
+  },
+  "keywords": [
+    "UI",
+    "kit",
+    "css"
+  ],
+  "author": "Arun Michael Dsouza <amdsouza92@gmail.com>",
+  "license": "MIT",
+  "homepage": "http://arunmichaeldsouza.github.io/CSS-Mint/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ArunMichaelDsouza/CSS-Mint.git",
+    "fileMap": [
+      {
+        "basePath": "build",
+        "files": [
+          "css-mint*"
+         ]
+       }
+     ]
```

Comment:
Can you please ident the closing `]` and `}` correctly? You can refer to an already existing package.json file such as [this one](https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/jsSocials/package.json).

**Note**: Ignore the `version` field in the sample package.json file I've linked to.